### PR TITLE
fahad made changes on fahadun.md file

### PR DESCRIPTION
**Pull request by fahadun**
added caption on the fahad.md file. merge changes with master.

- one 
- two

@junyucao 
@Quintanillaa

### CODE REVIEW

**Review by RaymondChu3388 — COMMENTED** [review]
ok

**Review by RaymondChu3388 — APPROVED** [review]
I am a learner

File: _posts/0000-01-02-fahadun.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+this is a comment.
```
Comment (junyucao) [inline _posts/0000-01-02-fahadun.md:6]: so original, wow im speachless

**Review by junyucao — COMMENTED** [review]
denied!

**Review by eldiben87 — APPROVED** [review]
That is a comment!

**Review by waqarpseg — APPROVED** [review]
Very very niceeee

**Comment by fahadun** [comment]
merge is complete.

**Pull request merged**